lib/pf/Switch/Meraki/MR_v2.pm

```diff
@@ -165,6 +165,62 @@ sub parseRequest {
     return ($nas_port_type, $eap_type, $client_mac, $port, $user_name, $nas_port_id, $session_id, $nas_port_id);
 }
 
+sub returnRadiusAccessAccept {
+    my ($self, $args) = @_;
+    my $logger = $self->logger;
+
+    $args->{'unfiltered'} = $TRUE;
+    my @super_reply = @{$self->SUPER::returnRadiusAccessAccept($args)};
```

Comment:
The issue here is the call of returnRadiusAccessAccept that return the psk attributes of the WLC.
We should cleanup the @super_reply ({Cisco-AVPair}->{psk} and psk-mode)